### DISCUSSION TITLE
FreshRSS on external attached drive?

### DISCUSSION THREAD

**Discussion by NatHarari** [FreshRSS/FreshRSS#5781] [Q&A]
I'm trying to get FreshRSS to run with apache2 on my Raspberry Pi. It runs fine and is currently in /usr/share as per the setup for Raspberry Pi listed here: https://pimylifeup.com/raspberry-pi-freshrss/
I can see the page from localhost with a very basic apache2 config. It sets up great.
However, I want to move my FreshRSS directory to an external (locally connected) drive which is at: /media/pi/TIMEMACHINE/FreshRSS
I tried moving it with sudo mv, and that went fine. Then I linked it in /usr/share with a Symlink, still pointing the apache2 root to /usr/share/FreshRSS
I set the the chown etc... to www-data in the /media/pi/TIMEMACHINE/FreshRSS
And I get a 403 error when I try to hit my site. I've been over this for the last 7 hours and I'm going to end up tearing my hair out.
If anyone can please let me know how to do this, it would be greatly appreciated.
Thanks!

**Comment by Frenzie** [comment]
Did you try restarting Apache 2 after these changes?

**Reply by NatHarari** [reply]
Absolutely. sudo systemctl restart apache2
No errors.
Also did a reload and restarted the whole server as well.

**Reply by Frenzie** [reply]
This /media/pi thing isn't FAT32 or exFAT or something that might cause weird issues?

**Reply by NatHarari** [reply]
It's ext4

**Reply by Alwaysin** [reply]
What about "Options FollowSymLinks" in your Apache conf?

**Reply by NatHarari** [reply]
Here's my freshrss.conf in my /etc/apache2/sites-available:

```
<VirtualHost *:80>
    ServerAdmin webmaster@example.com
    ServerName mydomain.org
    DocumentRoot /usr/share/FreshRSS
    Options FollowSymLinks

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined
    
    # Possible additional configurations:
    # You can add additional configurations for PHP, SSL, etc. here
</VirtualHost>

```

**Reply by Alkarex** [reply]
Can you access a static file such as https://freshrss.example.net/robots.txt or is that also returning a 403?

**Reply by NatHarari** [reply]
I'll have to get back to you on that. I'm not able to check until this evening. I will though and thank you for that suggestion. I will let you know. 👍

**Reply by NatHarari** [reply]
In answer to your question: Yes I still get a forbidden page. I tried: http://mydomain.org/robots.txt.
I'll try working on it some more. Tonight and tomorrow are a bit busy, so I'll see. This is taking up a huge amount of my time which I don't currently have, but will have again in the near future. Thank you for your patience with me.

**Reply by Alkarex** [reply]
Ok, so if the static file returns a 403, then it is unrelated to FreshRSS and solely due to an Apache / server configuration issue
